tests/integration/api/action/test_api_action_classification.py

```diff
@@ -0,0 +1,239 @@
+"""API Tests for Action Classification training"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import glob
+import os.path as osp
+import random
+import time
+import warnings
+from concurrent.futures import ThreadPoolExecutor
+from typing import Optional
+
+import numpy as np
+import pytest
+
+from otx.algorithms.action.tasks import ActionClsInferenceTask, ActionClsTrainTask
+from otx.algorithms.common.tasks.training_base import BaseTask
+from otx.api.configuration.helper import create
+from otx.api.entities.annotation import AnnotationSceneEntity, AnnotationSceneKind
+from otx.api.entities.dataset_item import DatasetItemEntity
+from otx.api.entities.datasets import DatasetEntity
+from otx.api.entities.image import Image
+from otx.api.entities.inference_parameters import InferenceParameters
+from otx.api.entities.metrics import Performance
+from otx.api.entities.model import ModelEntity
+from otx.api.entities.model_template import (
+    TaskType,
+    parse_model_template,
+    task_type_to_label_domain,
+)
+from otx.api.entities.resultset import ResultSetEntity
+from otx.api.entities.subset import Subset
+from otx.api.entities.task_environment import TaskEnvironment
+from otx.api.entities.train_parameters import TrainParameters
+from otx.api.usecases.tasks.interfaces.export_interface import ExportType
+from otx.api.utils.shape_factory import ShapeFactory
+from otx.cli.datasets import get_dataset_class
+from otx.cli.utils.io import generate_label_schema
+from tests.test_suite.e2e_test_system import e2e_pytest_api
+from tests.unit.api.test_helpers import generate_random_annotated_image
+
+DEFAULT_ACTION_TEMPLATE_DIR = osp.join("otx/algorithms/action/configs", "classification", "x3d")
```

Comment:
Currently, test for only `x3d` ?

Reply:
Yes, currently X3D is the only backbone for action classification